src/platforms/php/guides/symfony/config.mdx

```diff
@@ -64,6 +59,22 @@ All the possible configurations under the `options` key map directly to the corr
 you can read more about those in the [general configuration docs](../configuration/),
 and on the [PHP specific SDK docs](../config/).
 
+If you use Monolog, you can configure it by appending a `services` entry to your configuration file:
```

Comment:
I would explain this piece with a bit more detail:
```suggestion
If you use Monolog, you can use it to report events instead of the normal error listener approach. To do so, you must turn `register_error_listener` to `false` and configure our `Handler` as a service, like this:
```

Also, @ste93cry are you sure that this is all it's needed? Shouldn't we hook the handler somehow? With either `autoconfigure: true` or by changing Monolog's config?

Reply:
>are you sure that this is all it's needed?

Yes, I know there is some work going on in the [`symfony/monolog-bundle`](https://github.com/symfony/monolog-bundle) repository to support out-of-the-box the registration of our handler, but being a WIP work I don't know when it will land

Shouldn't we hook the handler somehow? With either `autoconfigure: true` or by changing Monolog's config?

We could register the handler ourselves in the DI container so that users don't have to, if the service is `private: true` it will be dropped during the compilation phase if it's not used and so users will not have one more unused service in their service container. However, I don't think it's worth doing it because:

- The Monolog configuration is too app-specific and we cannot hook into it because the ordering matters: in fact, even Symfony does not allow merging multiple `monolog` configuration nodes
- The handler has a few options which can be tuned (`$level` and `$bubble`), and if we register the service and the user wants to customize those parameters they will have to anyway override the service definition. Usually these parameters are what you set when registering an handler in the Monolog configuration, so the best DX experience we could provide is to support using our handler directly in the Monolog bundle

Reply:
@ste93cry sorry I posed the wrong question: I was asking if, in the context of this doc, we should instruct the user to do something else or if just by registering the service it will be hooked into Monolog automatically.

Reply:
Registering the service is not enough, you then need to use it in the configuration of the Monolog bundle. However, how and where you use it is dependant on the user configuration and what he wants to do. Maybe could we link to the Monolog documentation for this part?

Reply:
We definitely need to tell the user that adding the handler to the Monolog config is required.